doc/api/vm.markdown

```diff
@@ -301,6 +301,34 @@ e.g. `(0,eval)('code')`. However, it also has the following additional options:
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
 
+## Example: Run a Server within a VM
```

Comment:
I think we are using the heading style when it is an example, outside of a method, but rather a guide or such. E.g. here https://github.com/nodejs/node/blob/master/doc/api/readline.markdown#example-tiny-cli . If you'd rather want it belonging to `.runInThisContext()`, I can do it w/o pounds.

Having a look at for example `streams`, it would be 3rd level, but because it's having 2nd level topics like `Writable` etc. We are a little inconsistent about this I fear. :(

No strong opinions though.